osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -225,13 +225,12 @@ public void PopulateUser(User user, Action success = null, Action failure = null
                 success?.Invoke();
             };
 
-            req.Failure += _ =>
-            {
-                user.Id = 1;
```

Comment:
have you checked the blame for this? it looks to be intentional.

Reply:
Seems to have came from this commit https://github.com/ppy/osu/commit/5d96e6d90a30b85d27866d70ce9ff5f693247e0f, I guess it makes sense for the team editor screen to force ID `1` on invalid.